srcalc/src/test/java/gov/va/med/srcalc/web/DynamicValueVisitorTest.java

```diff
@@ -0,0 +1,86 @@
+package gov.va.med.srcalc.web;
+
+import static org.junit.Assert.*;
+import gov.va.med.srcalc.domain.SampleObjects;
+import gov.va.med.srcalc.domain.variable.BooleanValue;
+import gov.va.med.srcalc.domain.variable.BooleanVariable;
+import gov.va.med.srcalc.domain.variable.DiscreteNumericalValue;
+import gov.va.med.srcalc.domain.variable.DiscreteNumericalVariable;
+import gov.va.med.srcalc.domain.variable.MultiSelectOption;
+import gov.va.med.srcalc.domain.variable.MultiSelectValue;
+import gov.va.med.srcalc.domain.variable.MultiSelectVariable;
+import gov.va.med.srcalc.domain.variable.NumericalRange;
+import gov.va.med.srcalc.domain.variable.NumericalValue;
+import gov.va.med.srcalc.domain.variable.NumericalVariable;
+import gov.va.med.srcalc.domain.variable.ProcedureValue;
+import gov.va.med.srcalc.domain.variable.ProcedureVariable;
+import gov.va.med.srcalc.web.view.VariableEntry;
+
+import org.junit.Before;
+import org.junit.Test;
+
+
+/**
+ * Tests {@link DynamicValueVisitor}.
+ */
+public class DynamicValueVisitorTest
+{
+	private DynamicValueVisitor fVisitor;
+	
+	@Before
+	private void visitorSetup()
+	{
+		final VariableEntry variableEntry = new VariableEntry(null);
```

Comment:
You triggered my null aversion! *Barf.* How about passing an empty collection instead?